packages/dpns-contract/schema/dpns-contract-documents.json

```diff
@@ -42,18 +42,25 @@
       },
       "normalizedLabel": {
         "type": "string",
-        "pattern": "^[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$",
+        "pattern": "^[a-km-z1-9][a-km-z1-9-]{0,61}[a-km-z1-9]$",
```

Comment:
Wrong pattern: "Base58 excludes zero, uppercase 'O', uppercase 'I', and lowercase 'l'". Hence this is wrong, you did not exclude i.

Reply:
I'm not following. This pattern doesn't allow upper case at all why should we exclude lower case i?

Reply:
Base58 excludes uppercase I. when we normalize a name like Ivan it would become ivan. Ivan and 1van collide.